packages/node/src/edge-functions/edge-handler.ts

```diff
@@ -192,24 +191,31 @@ export async function createEdgeEventHandler(
       process.exit(1);
     }
 
-    const response = await fetch(server.url, {
+    const headers = new Headers(request.headers as HeadersInit);
```

Comment:
Got this error testing an edge function that uses a WASM module using Node 14. Node 16 works.

```
import init, { greet } from "../wasm/pkg";
//@ts-ignore
import wasmModule from "../wasm/pkg/wasm_bg.wasm?module";

export const config = {
	runtime: "experimental-edge",
};

export default async function handler() {
	await init(wasmModule);
	const helloWorld = greet();
	return new Response(helloWorld);
}
```

```
TypeError: undici_1.Headers is not a constructor
    at /Users/chris/code/vercel/packages/node/dist/edge-functions/edge-handler.js:146:25
    at Server.onDevRequest (/Users/chris/code/vercel/packages/node/dist/dev-server.js:85:49)
    at Server.emit (events.js:400:28)
    at Server.emit (domain.js:475:12)
    at parserOnIncoming (_http_server.js:900:12)
    at HTTPParser.parserOnHeadersComplete (_http_common.js:127:17)%
```

Reply:
I think there was a breaking change for fetch in Undici between those node versions. We may need to keep node-fetch around unless we drop Node 14 support.

Reply:
Confirmed reverting to `node-fetch` has resolved the error.